src/views/ServerStartView.vue

```diff
@@ -2,21 +2,50 @@
   <div
     class="font-sans flex flex-col justify-center items-center h-screen m-0 text-neutral-300 bg-neutral-900 dark-theme pointer-events-auto"
   >
-    <h2 class="text-2xl font-bold">{{ ProgressMessages[status] }}</h2>
+    <div
+      v-if="status == ProgressStatus.ERROR"
+      class="flex flex-col items-center my-[60px]"
+    >
+      <h1 class="text-[60px] my-0">{{ $t('serverStartView.errorTitle') }}</h1>
+      <h2 class="my-[30px]">{{ $t('serverStartView.errorDescription') }}</h2>
+      <Button
+        class="bg-[#059669] border-[#059669] text-white"
+        icon="pi pi-refresh"
+        :label="t('serverStartView.reinstall')"
+        @click="reinstall"
+      />
+
+      <div class="flex gap-[30px] pt-[10px]">
+        <Button
+          variant="text"
```

Comment:
`variant="outline"` should do equivalent.

Reply:
Unfortunately it doesn't (see updated screenshot)